pkg/gardenlet/apis/config/types.go

```diff
@@ -457,6 +457,11 @@ type ETCDConfig struct {
 	BackupCompactionController *BackupCompactionController
 	// BackupLeaderElection contains configuration for the leader election for the etcd backup-restore sidecar.
 	BackupLeaderElection *ETCDBackupLeaderElection
+	// FeatureGates is a map of feature names to bools that enable or disable alpha/experimental
+	// features. This field modifies piecemeal the built-in default values from
+	// "github.com/gardener/etcd-druid/pkg/features/features.go".
+	// Default: nil
```

Comment:
Do we need to add `// +optional` here?

Reply:
Nope, that's the internal type. We add it only to the external ones.